### PR TITLE
FRED-628

### PR DESCRIPTION
**Pull request by scmonheim**
# FRED-628 ESS-ESS and ESS-DSS API initial draft

This draft is an initial cut at the ESS-ESS/DSS API, initially built off of two specs, the UTM ASTM spec v1.0, and the UAM-SP X4 spec

### CODE REVIEW

File: etm/ess-interface-spec/etm-ess-api.yaml [hunk 1]
```diff
@@ -2086,6 +2088,197 @@ components:
...
+      externalDocs:
+        description: See FIXM 4.2.0 for further information.
+        url: https://fixm.aero/releases/FIXM-4.2.0/doc/schema_documentation/Fixm_AircraftRegistrationType.html#LinkBC
+    Operator:
+      type: object
+      description: >-
+        The information about an operator, identifier, email, telephone number
+      properties:
+        operator_name:
+          $ref: '#/components/schemas/OperatorName'
```
Comment (ewahl-NASA) [inline etm/ess-interface-spec/etm-ess-api.yaml:2137]: I believe this is supposed to be a free-form text field, of max length 256.

File: etm/ess-interface-spec/etm-ess-api.yaml [hunk 2]
```diff
@@ -2086,6 +2088,197 @@ components:
...
+        by which the operation will interact with other operations by default. Does not preclude
+        operator-operator negotiation or agreements @todo better description @spencer
+      default: 1
+    NegotiatedOperators:
+      type: array
+      description: >-
+        List of other operators that the operator of this operation has agreements with for the
+        purposes of negotiation outside of the common operating procedures.
+      items:
+        $ref: '#/components/schemas/OperatorName'
```
Comment (ewahl-NASA) [inline etm/ess-interface-spec/etm-ess-api.yaml:2184]: This is supposed to be an array (max length of 1024) of free-format strings (of max length 256).

File: etm/ess-interface-spec/etm-ess-api.yaml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 openapi: 3.0.2
 info:
-  title: UTM API (USS->DSS and USS->USS)
+  title: ETM API (ESS->DSS and ESS->ESS)
   version: 1.0.0
   description: >-
-    Interface definitions for communication between a UAS Service Supplier (USS)
-    and a Discovery and Synchronization Service (DSS), as well as for
-    communication between two USSs.
+    Interface definitions for communication between a ESS
```
[the remark this answers is not in the document]
Comment (issmith1) [reply etm/ess-interface-spec/etm-ess-api.yaml:6]: upvote

**Comment by ewahl-NASA** [comment]
One other thing: I suggest changing the name of this file. The name isn't really accurate, and being the same name as the ETM Operator API spec is kind of confusing.

**Pull request merged**